### PR TITLE
adding template for web app and sql database

### PR DESCRIPTION
**Pull request by tfitzmac**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@tfitzmac__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (tomfitz). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@davidebbo Can you sign off on this? Thanks!

File: app-service-web-arm-with-sql-database-provision/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+      "name": "[parameters('siteName')]",
+      "type": "Microsoft.Web/Sites",
+      "location": "[parameters('siteLocation')]",
+      "dependsOn": ["[concat('Microsoft.Web/serverFarms/', parameters('hostingPlanName'))]"],
+      "tags": {
+        "[concat('hidden-related:', resourceGroup().id, '/providers/Microsoft.Web/serverfarms/', parameters('hostingPlanName'))]": "empty"
+      },
+      "properties": {
+        "name": "[parameters('siteName')]",
+        "serverFarm": "[parameters('hostingPlanName')]"
```
Comment (davidebbo) [inline app-service-web-arm-with-sql-database-provision/azuredeploy.json]: This is the old API. In newer one, prop is called serverFarmID.

Comment (davidebbo) [inline app-service-web-arm-with-sql-database-provision/azuredeploy.json]: See https://github.com/davidebbo/AzureWebsitesSamples/blob/master/ARMTemplates/WebAppManyFeatures.json for example of newest API use.

**Pull request merged**